examples/graph/app.js

```diff
@@ -162,34 +163,51 @@ class Root extends Component {
     }
   }
 
-  _onDragMove(el) {
+  _onMouseDown(event) {
+    // Use DeckGL.queryObject() to find the object under the mouse,
+    // and store it for updating on mouse move.
+    const point = {x: event.clientX, y: event.clientY};
+    const el = this.deckGL.queryObject(point);
```

Comment:
Nit: Please type out the name of variables (e.g. `el`). We have called this object `info` or `pickInfo` in other places, not sure if that makes sense any more `queryInfo` perhaps.